src/encoder.rs

```diff
@@ -2322,4 +2373,50 @@ mod test {
     assert_eq!(RAV1E_PARTITION_TYPES[RAV1E_PARTITION_TYPES.len() - 1],
                PartitionType::PARTITION_SPLIT);
   }
-}
+
+  fn create_frame(frame_width: usize, frame_height: usize) -> (Frame<u16>, FrameInvariants<u16>) {
+    let mut frame = Frame::<u16>::new(frame_width, frame_height, ChromaSampling::Cs420);
```

Comment:
Is it really correct to hardcode 4:2:0 here?

Reply:
Not really. Since we're dealing with only luma plane here, I think the chroma subsampling is irrelevant in this test.

Reply:
OK, make sense.